src/components/MAPI/utils.ts

```diff
@@ -1093,6 +1093,8 @@ export function getClusterDescription(
 
   const { kind, apiVersion } = infrastructureRef;
   switch (true) {
+    case kind === capav1beta1.AWSCluster &&
+      apiVersion === capav1beta1.ApiVersion:
```

Comment:
Since the cluster description is an annotation on the Cluster resource, and we use the same Cluster CR for all providers, what do you think about making this case the default behaviour? That way we could avoid having a long list of cases here as we support more CAPI providers